cmd/lotus-storage-miner/storage.go

```diff
@@ -103,3 +112,192 @@ var storageAttachCmd = &cli.Command{
 		return nodeApi.StorageAddLocal(ctx, p)
 	},
 }
+
+var storageListCmd = &cli.Command{
+	Name:  "list",
+	Usage: "list local storage paths",
+	Action: func(cctx *cli.Context) error {
+		nodeApi, closer, err := lcli.GetStorageMinerAPI(cctx)
+		if err != nil {
+			return err
+		}
+		defer closer()
+		ctx := lcli.ReqContext(cctx)
+
+		st, err := nodeApi.StorageList(ctx)
+		if err != nil {
+			return err
+		}
+
+		local, err := nodeApi.StorageLocal(ctx)
+		if err != nil {
+			return err
+		}
+
+		for id, sectors := range st {
+			var u, s, c int
+			for _, decl := range sectors {
+				if decl.SectorFileType&sectorbuilder.FTUnsealed > 0 {
```

Comment:
this feels hacky, can't we just have a method that turns this into a sector file type, and we can switch on it?